l10n_br_resource/models/resource_calendar.py

```diff
@@ -0,0 +1,263 @@
+        if quantidade_dias > 30:
+            return 30
+        else:
+            return quantidade_dias
+
+    @api.multi
+    def data_eh_feriado_bancario(self, data_referencia=datetime.now()):
+        """Verificar se uma data é feriado bancário.
+        :param datetime data_referencia: Se nenhuma data referencia for passada
+                                    verifique se hoje é feriado bancário.
+                                    Se a data referencia for passada, verifique
+                                    se a data esta dentro de algum leave
+                                    date_start <= data_referencia <= data_end
+
+        :return int leaves_count: +1 se for feriado bancário
+                                   0 se a data nao for feriado bancário
+        """
+        domain = [
+            ('date_from', '<=', data_referencia.strftime("%Y-%m-%d %H:%M:%S")),
+            ('date_to', '>=', data_referencia.strftime("%Y-%m-%d %H:%M:%S")),
+            ('leave_type', 'in', ['F', 'B']),
+        ]
+        leaves_count = \
+            self.env['resource.calendar.leaves'].search_count(domain)
+        return leaves_count
+
+    @api.multi
+    def data_eh_feriado_emendado(self, data_referencia=datetime.now()):
+        """Verificar se uma data é feriado emendado.
+        :param datetime data_referencia: Se nenhuma data referencia for passada
+                                   verifique se hoje é feriado emendado.
+                                   Se a data referencia for passada, verifique
+                                   se a data esta dentro de algum leave
+                                   date_start <= data_referencia <= data_end
+
+        :return retorna True ou False
+        """
+        dia_antes = data_referencia - timedelta(days=2)
+        dia_depois = data_referencia + timedelta(days=2)
+
+        dia_antes_eh_util = \
+            True if dia_antes.weekday() > 5 or self.data_eh_feriado(
+                dia_antes) else False
+        dia_depois_eh_util = \
+            True if dia_depois.weekday() > 4 or self.data_eh_feriado(
+                dia_depois) else False
+
+        return dia_antes_eh_util or dia_depois_eh_util
+
+
+class ResourceCalendarLeave(models.Model):
+
+    _inherit = 'resource.calendar.leaves'
+
+    country_id = fields.Many2one(
+        'res.country', string=u'País',
+        related='calendar_id.country_id',
+    )
+    state_id = fields.Many2one(
+        'res.country.state', u'Estado',
+        related='calendar_id.state_id',
+        domain="[('country_id','=',country_id)]",
+        readonly=True
+    )
+    l10n_br_city_id = fields.Many2one(
+        'l10n_br_base.city', u'Municipio',
+        related='calendar_id.l10n_br_city_id',
+        domain="[('state_id','=',state_id)]",
+        readonly=True
+    )
+    leave_type = fields.Selection(
+        string=u'Tipo',
+        selection=[item for item in TIPO_FERIADO.iteritems()],
```

Comment:
> `selection=[item for item in TIPO_FERIADO.iteritems()],`

Can be also written as:

    selection=list(TIPO_FERIADO.iteritems()),

Or simply (in Python 2 at least):

    selection=TIPO_FERIADO.items(),